react/client/src/styles/utilityStyles.ts

```diff
@@ -1,7 +1,25 @@
-import { makeStyles, createStyles } from '@material-ui/core';
+import { Theme, makeStyles, createStyles } from '@material-ui/core';
 
-const useUtilityStyles = makeStyles((theme) =>
+interface IStyleProps {
+  pageTheme?: string;
+  isSoloContainer?: boolean;
+}
+
+const useUtilityStyles = makeStyles<Theme>((theme) =>
```

Comment:
Heavily leaning on theming now. It can cover an entire page of different styles as we pass in a different `pageTheme`.